shared_libraries/orca_shared/database/use_cases/validation.py

```diff
@@ -27,6 +26,18 @@ def _validate_password(password: str, context: str, logger: logging.Logger) -> N
         msg = f"{context} password must be at least 12 characters long."
         logger.critical(msg)
         raise Exception(msg)
+    elif re.search('[0-9]', password) is None:
+        msg = f"{context} password must contain a digit between 0 and 9."
+        logger.critical(msg)
+        raise Exception(msg)
+    elif re.search('[A-Z]', password) is None:
+        msg = f"{context} password must contain an upper case letter."
+        logger.critical(msg)
+        raise Exception(msg)
+    elif re.search('[$&%+,:;=?.*^@#]', password) is None:
+        msg = f"{context} password must contain a special character."
+        logger.critical(msg)
+        raise Exception(msg)
```

Comment:
Good list of checks. And db_deploy will run these during deployment, so this will raise errors during deployment as well.

Reply:
yes it will fail. Added steps to migration notes.